.github/workflows/bundle_with_dakota_macos.yml

```diff
@@ -0,0 +1,283 @@
+        
+        libgccpath=$(dirname $(find /usr/local/Cellar/gcc -name libgcc_s.1.1.dylib 2>/dev/null | head -n 1))
+        libdakotapath=$(dirname $(find ${HOME}/${INSTALL_DIR}/lib -name libdakota_util.dylib 2>/dev/null | head -n 1))
+
+        echo "libgccpath=$libgccpath"
+        echo "libdakotapath=$libdakotapath"
+
+        DYLD_LIBRARY_PATH="$DYLD_LIBRARY_PATH:$libgccpath:libdakotapath"
+
+        install_name_tool -add_rpath "$libgccpath" "$carolina_so_path"
+        install_name_tool -add_rpath "${HOME}/${INSTALL_DIR}/lib" "$carolina_so_path"
+
+        mkdir /tmp/carolina_dist
+        mkdir /tmp/carolina_dist_unfixed
+        pip install build
+        python -m build --wheel . --outdir /tmp/carolina_dist_unfixed
+        echo $carolina_so_path
+        otool -l $carolina_so_path
+        echo "Finding missing dylibs"
+
+        ls -lh /tmp/carolina_dist_unfixed
+        pv=$(echo ${{ matrix.python-version }} | sed 's/\.//g')
+        unfixed_wheel_path=$(find /tmp/carolina_dist_unfixed -name "carolina*$pv*whl")
+        echo "Found Carolina wheel at $unfixed_wheel_path"
+        pip install delocate
+        delocate-listdeps $unfixed_wheel_path
+        delocate-wheel -w /tmp/carolina_dist -v $unfixed_wheel_path 
+        echo "Output new carolina wheel to /tmp/carolina_dist"
+        ls -lh /tmp/carolina_dist
+    - uses: actions/upload-artifact@v3
+      id: cache-carolina_wheel
+      with:
+        path: |
+          /tmp/carolina_dist/carolina*
+
+  tests:
+    needs: build_dependencies
+    if: ${{ needs.build_dependencies.result != 'failed' }}
+    name: "tests_${{ matrix.os }}_py-${{ matrix.python-version }}_${{ matrix.test_type }}_ert-${{ matrix.ert_version }}"
+    env:
+      ERT_SHOW_BACKTRACE: ${{ inputs.ERT_SHOW_BACKTRACE }}
+      NO_PROJECT_RES: ${{ inputs.NO_PROJECT_RES }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ '3.8','3.9','3.10' ]
+        os: ['macos-latest']
+        test_type: ['doc', 'test', 'integration-test']
+        ert_version: ['release', 'main']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: "pip"
+        cache-dependency-path: |
+          setup.py
+          test_requirements.txt
+          requirements.txt
+    - name: Download Carolina artifact
+      uses: actions/download-artifact@v3
+      with:
+        path: /tmp/artifacts
+    - name: ls the artifacts dir
+      run: ls /tmp/artifacts
+    - name: Find the Carolina wheel
+      run: |
+        pyver_nodot=$(echo ${{ matrix.python-version }} | sed 's/\.//g')
+        wheel_path=$(find /tmp/artifacts -name "carolina*$pyver_nodot*macos*whl")
+        echo "Found Carolina wheel at $wheel_path"
+        ls -lh $wheel_path
```

Comment:
Remove?

Reply:
Removed the ls -lh, I think keeping the `echo "Found Carolina wheel at $wheel_path"` for monitoring purposes makes sense in case this breaks at some point in the future.